<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listFiles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>501</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>411</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="grpFilter">
       <property name="title">
        <string>Filter</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>171</width>
          <height>401</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkSoftware">
           <property name="text">
            <string>Software</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkSize">
           <property name="text">
            <string>Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkDate">
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkISO">
           <property name="text">
            <string>ISO Speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkExpo">
           <property name="text">
            <string>Exposure Time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkAltitude">
           <property name="text">
            <string>GPS Altitude</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>479</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnOpen">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDetail">
       <property name="text">
        <string>Detail</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
